app/src/main/res/values/strings.xml

```diff
@@ -276,8 +277,8 @@
     <string name="keyboard">Keyboard</string>
     <string name="magic_keyboard_title">Magikeyboard</string>
     <string name="magic_keyboard_summary">Activate a custom keyboard populating your passwords and all identity fields.</string>
-    <string name="magic_keyboard_preference_title">Magikeyboard settings</string>
-    <string name="magic_keyboard_configure_title">How to configure the keyboard for secure form filling?</string>
+    <string name="magic_keyboard_preference_title">Magikeyboard Settings</string>
+    <string name="magic_keyboard_configure_title">Set up the keyboard to autofill forms securely.</string>
```

Comment:
It's not an autofill

Reply:
In fact yes, it's just that I differentiate the "Autofill" feature from the "keyboard" feature. It's good like that.